cli/README.md

```diff
@@ -154,3 +154,11 @@ Once you've tested an update to a specific deployment, and you want to promote i
 code-push promote <appName> <sourceDeploymentName> <destDeploymentName>
 code-push promote MyApp Staging Production
 ```
+
+### Viewing release history
+
+You can view a history of your releases using the following command:
```

Comment:
What about something like... "In order to view the history of releases for a specific app deployment (including promotions), you can use the following command"